django_app/redbox_app/redbox_core/models.py

```diff
@@ -740,19 +765,29 @@ class Citation(UUIDPrimaryKeyBase, TimeStampedModel):
     class Origin(models.TextChoices):
         WIKIPEDIA = "Wikipedia", _("wikipedia")
         USER_UPLOADED_DOCUMENT = "UserUploadedDocument", _("user uploaded document")
-        GOV_UK = "GOV.UK", _("gov.uk")
+        GOV_UK = "Gov.Uk", _("gov.uk")
```

Comment:
Yeah this is a bit odd, but is actually required because of the `.title()` here https://github.com/i-dot-ai/redbox/blob/f224d203f6fd1e4b2ce4ab8dd02edff2a22e78f7/django_app/redbox_app/redbox_core/consumers.py#L219-L220